examples/component-scripts/zarf.yaml

```diff
@@ -4,14 +4,23 @@ metadata:
   description: "Test component to demonstrate script timeout feature"
 
 components:
+  - name: prepare
+    required: true
+    scripts:
+      # Note these files will be created in this directory, regardless of where the package create command is called from
```

Comment:
Adding this as a comment since it is so minor, but there is only one file here. (this file)